client/jetpack-cloud/sections/partner-portal/revoke-license-dialog/index.tsx

```diff
@@ -15,15 +15,17 @@ interface Props {
 	product: string;
 	siteUrl: string | null;
 	onClose: ( action?: string ) => void;
-	isChildLicense?: boolean;
+	licenseRole?: 'parent' | 'child' | 'single';
```

Comment:
Nitpick: What do you think about defining these as constants instead of using single values? It's a more defensive approach as you can easily do a typo when writing the single values, and also it's much easier for refactoring. Definitely a non-blocker, I'd suggest creating a follow-up maintenance card to improve this if you think it makes sense.